package.json

```diff
@@ -34,12 +35,15 @@
     "index.es.js"
   ],
   "dependencies": {
-    "node-sass": "4",
     "rollup-pluginutils": "2"
   },
   "devDependencies": {
     "rollup": "1",
     "rollup-plugin-buble": "0",
     "standard": "14"
+  },
+  "peerDependencies": {
```

Comment:
`sass` should probably just be `devDependencies`, so it doesn't throw an error from the package manager if it's missing.

Reply:
Then `node-sass` can be an actual dependency so it can default to it.

Reply:
If we keep node-sass as a dependency then this will be a minor version.
If node-sass needs to be installed separately (more efficient) then this is a breaking change => major version.

I don't mind either (not actively using this package anymore).